UtilizeSkylineDataServices/Readme.md

```diff
@@ -23,104 +23,145 @@ From the panel of the web app, you will be able to:
+-	[Skyline API Key Support](http://www.systemlinkcloud.com/api-download) for SystemLink Cloud
+
+If you do not install LabVIEW NXG 2.1 Web Module, NI Package Manager will not install the Skyline API Key Support package on to your machine. You will receive an installation dependency error as a result. The Skyline API Key Support package enables you to establish a connection to the SystemLink Cloud server. Please refer to [Skyline API Support](https://www.systemlinkcloud.com/api-download) for installation instructions and the [FAQ](https://www.systemlinkcloud.com/faq) for help navigating installation errors.
 
 # Choosing Your Goal
 Do you want to learn about connecting your web application to SystemLink Cloud or an on-premises server? Click one of the following links to jump to your workflow:
 
--	[I want to connect to the SystemLink Cloud](https://github.com/ni/webvi-examples/blob/master/UtilizeSkylineDataServices/Readme.md#systemlink-cloud).
+-	[I want to install the Skyline API Key Support package](https://github.com/ni/webvi-examples/blob/master/UtilizeSkylineDataServices/Readme.md#skyline-api-key-support-package).
+- [I want to connect to SystemLink Cloud](https://github.com/ni/webvi-examples/blob/master/UtilizeSkylineDataServices/Readme.md#systemlink-cloud).
 - [I want to connect to an on-premises web server](https://github.com/ni/webvi-examples/blob/master/UtilizeSkylineDataServices/Readme.md#on-premises-web-server).
 -	[I want to learn about the details involved in creating the WebVI](https://github.com/ni/webvi-examples/blob/master/UtilizeSkylineDataServices/Readme.md#webvi-details).
 -	[I want to learn about security](https://github.com/ni/webvi-examples/blob/master/UtilizeSkylineDataServices/Readme.md#security).
 
+# Skyline API Key Support Package
+
+Prior to writing code, [download](https://www.systemlinkcloud.com/api-download) the Skyline API Key Support package to enable you to establish a connection with SystemLink Cloud from your WebVI. Depending on your version of LabVIEW or LabVIEW NXG 2.1 Web Module, follow the instructions below.
+
+## LabVIEW NXG 2.1 Web Module:
+
+1.	Go to [API Key Support](https://www.systemlinkcloud.com/api-download).
+2.	Install LabVIEW NXG 2.1 Web Module.
+>**Note:** You must install LabVIEW NXG 2.1 Web Module for the Skyline API Support Key package to install on your machine.
+
+3.	Click **Download Skyline API Key Support**.
+4.	Double-click the package file to launch NI Package Manager.
+5.	Follow instructions in the installation prompt to complete installation.
+
+Once you successfully install the Skyline API Key Support package, you can connect your WebVIs to SystemLink Cloud from LabVIEW NXG 2.1 Web Module.
+
+## LabVIEW 2015-2017:
+
+1.	[Install](https://www.systemlinkcloud.com/api-download) LabVIEW NXG 2.1 Web Module.
+>**Note:** You must install LabVIEW NXG 2.1 Web Module for the Skyline API Support Key package to install on your machine.
```

Comment:
We can remove some of these instructions. Installing the Web Module gives you everything you need to access SystemLink Cloud from CG LV.  The API Key support package is for NXG only.